src/evo/providertx.cpp

```diff
@@ -187,10 +190,13 @@ std::string CProUpServTx::ToString() const
     }
 
     return strprintf("CProUpServTx(nVersion=%d, nType=%d, proTxHash=%s, operatorPayoutAddress=%s, "
-                     "platformNodeID=%s, platformP2PPort=%d, platformHTTPPort=%d)\n"
+                     "platformNodeID=%s%s)\n"
                      "  %s",
```

Comment:
nit: consider merging 2 lines for simplicity

```diff
-                     "platformNodeID=%s%s)\n"
-                     "  %s",
+                     "platformNodeID=%s%s)\n   %s",
```

Reply:
Log format was changed to be single-line, realigned with the remaining set of printed values, resolved in latest push.